fem/libceed_diffusion.qf

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
Could we move these `qf` files to the `ceed` subdir?

Reply:
That's a mistake, I should have removed them.